src/seedu/addressbook/data/person/Phone.java

```diff
@@ -56,4 +56,9 @@ public int hashCode() {
     public boolean isPrivate() {
         return isPrivate;
     }
+
+    @Override
+    public String getPrintableString() {
+        return "Phone: " + this.value;
```

Comment:
👍  for adding "Phone" here, increase readability. Applicable to other classes as well.